projects/plugins/jetpack/extensions/blocks/revue/edit.js

```diff
@@ -6,7 +8,12 @@ import './view.scss';
 
 // eslint-disable-next-line jsdoc/require-jsdoc
 export default function RevueEdit( { className } ) {
-	const migrationLink = 'https://wordpress.com/go/digital-marketing/migrate-from-revue-newsletter/';
+	const migrationMoreInfoLink = getRedirectUrl(
+		'https://wordpress.com/go/digital-marketing/migrate-from-revue-newsletter/'
+	);
+	const migrationLink = getRedirectUrl(
+		`https://wordpress.com/people/email-followers/${ getSiteFragment() }`
+	);
```

Comment:
Sorry, I should have been clearer. You would need to create the redirect in MC, and use the redirect slug here. You can learn more about this here:
PCYsg-pY7-p2

Reply:
Got it, thanks @jeherve . I wasn't 100% sure about the WordPress.com dashboard redirect as adding the site URL results in a URL like `https://wordpress.com/people/email-followers/reduced-plover.jurassic.ninja?site=reduced-plover.jurassic.ninja` instead of `https://wordpress.com/people/email-followers/reduced-plover.jurassic.ninja`, however it still works. Is this expected?

Reply:
Yep, the `site=` string is expected.